doc/source/development/contributing.rst

```diff
@@ -930,8 +930,12 @@ The ``pandas.util.testing`` module has many special ``assert`` functions that
 make it easier to make statements about whether Series or DataFrame objects are
 equivalent. The easiest way to verify that your code is correct is to
 explicitly construct the result you expect, then compare the actual result to
-the expected correct result::
+the expected correct result
```

Comment:
period at the end
"writting" --> "writing"
"#Issue number ##NUMBER:" --> "# See GH#NUMBER"

Reply:
Alterations are correct